velox/vector/tests/VectorTestBase.h

```diff
@@ -18,10 +18,13 @@
 #include "velox/vector/FlatVector.h"
 #include "velox/vector/tests/VectorMaker.h"
 
+#include <gtest/gtest.h>
+
 namespace facebook::velox::test {
 
 /// Returns indices buffer with sequential values going from size - 1 to 0.
 BufferPtr makeIndicesInReverse(vector_size_t size, memory::MemoryPool* pool);
+void assertEqualVectors(const VectorPtr& expected, const VectorPtr& actual);
```

Comment:
This function already exists in multiple places. It would be nice to consolidate these into just one.

RowContainerTest
OperatorTestBase
EvalSimplifiedTest
ExprTest
FunctionBaseTest
UnsafeRowVectorDeserializerTest